typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -86,4 +186,610 @@ export class EvmIsmModule extends HyperlaneModule<
       config,
     });
   }
+
+  protected async updateRoutingIsm(params: {
+    destination: ChainName;
+    config: RoutingIsmConfig;
+    origin?: ChainName;
+    mailbox?: Address;
+    existingIsmAddress: Address;
+    logger: Logger;
+  }): Promise<IRoutingIsm> {
+    const { destination, config, mailbox, existingIsmAddress, logger } = params;
+
+    // 1. Filter for known domains
+    const { domains, safeConfigDomains } = EvmIsmModule.filterRoutingIsmDomains(
+      { config, multiProvider: this.multiProvider },
+    );
+    config.domains = domains;
+
+    // 2. Compute the delta
+    const delta: RoutingIsmDelta = await routingModuleDelta(
+      destination,
+      existingIsmAddress,
+      config,
+      this.multiProvider,
+      this.factories,
+      mailbox,
+    );
+
+    // Step 3: Get owner and signer
+    const signer = this.multiProvider.getSigner(destination);
+    const provider = this.multiProvider.getProvider(destination);
+    const owner = await DomainRoutingIsm__factory.connect(
+      existingIsmAddress,
+      provider,
+    ).owner();
```

Comment:
would expect contracts to already be connected when the module.update is called

Reply:
since we're not getting the calldata from the interface - no longer need to connect contracts for an update